include/pipettes/core/interfaces.hpp

```diff
@@ -96,4 +126,20 @@ template <>
 auto motor_configurations<PipetteType::NINETY_SIX_CHANNEL>()
     -> HighThroughputMotorConfigurations;
 
-}  // namespace interfaces
\ No newline at end of file
+template <>
+auto sensor_configurations<PipetteType::SINGLE_CHANNEL>()
```

Comment:
Mind grouping the generic and specific templates next to each other for readability purposes?